Dockerfile

```diff
@@ -33,10 +33,10 @@ WORKDIR /src/
 
 RUN set -x \
     && pip install -U pip setuptools wheel \
-    && pip install -r /tmp/requirements/tests.txt \
+    && pip install -r /tmp/requirements/test.txt \
     && pip install -r /tmp/requirements/deploy.txt \
                    -r /tmp/requirements/main.txt \
-    && pip install -e . \
+    && pip install . \
```

Comment:
This needs to stay `-e` for development mounting purposes.

Reply:
Ok :+1: , let me see if I can get it to work.

Reply:
If you can't get this to work out (though please do try) we will at the very least be resolving this issue in the near future by removing the circular dependency. That is when the triggers have been moved to this package (i.e. #39 ).

Reply:
I added a `pip uninstall` before `pip install`.  It's not ideal but it works.